nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -34,13 +50,128 @@ define([
+                }).then(() => {                    
+                    layout.events.attachEvents(container);
+                });
+
+            });          
+        
+        }
+
+
+        /*
+            Options: {
+                workspaceInfo: workspace information
+                node: container node to build the widget into
+            }
+
+            This is more or less copied over from the way that the appCell handles building the widget. This model assumes one instance of the params widget per cell, so may need some adjustment as we make the bulk import work with multiple data types
+        */
+        function buildParamsWidget(options) {
+            const bus = runtime.bus().makeChannelBus({ description: 'Parent comm bus for input widget' });
+
+            const workspaceInfo = options.workspaceInfo,
+                node = options.node;
+
+            const widget = ParamsWidget.make({
+                bus: bus,
+                workspaceInfo: workspaceInfo,
+                initialParams: model.getItem('params')
+            });
+
+            bus.on('sync-params', function(message) {
+                message.parameters.forEach(function(paramId) {
+                    bus.send({
+                        parameter: paramId,
+                        value: model.getItem(['params', message.parameter])
+                    }, {
+                        key: {
+                            type: 'update',
+                            parameter: message.parameter
+                        }
+                    });
+                });
+            });
+
+            bus.on('parameter-sync', function(message) {
+                var value = model.getItem(['params', message.parameter]);
+                bus.send({
+                    value: value
+                }, {
+                    // This points the update back to a listener on this key
+                    key: {
+                        type: 'update',
+                        parameter: message.parameter
+                    }
+                });
+            });
+
+            bus.on('set-param-state', function(message) {
+                model.setItem('paramState', message.id, message.state);
+            });
+
+            bus.respond({
+                key: {
+                    type: 'get-param-state'
+                },
+                handle: function(message) {
+                    return {
+                        state: model.getItem('paramState', message.id)
+                    };
+                }
+            });
+
+            bus.respond({
+                key: {
+                    type: 'get-parameter'
+                },
+                handle: function(message) {
+                    return {
+                        value: model.getItem(['params', message.parameterName])
+                    };
+                }
             });
+
+            //TODO: disabling for now until we figure out what to do about state
```

Comment:
this is one TODO i actually put in, should probably be part of that state management ticket we have